src/wizard/dotnet/aspnet.md

```diff
@@ -0,0 +1,64 @@
+---
+name: ASP.NET
+doc_link: https://docs.sentry.io/platforms/dotnet/guides/aspnet/
+support_level: production
+type: framework
+---
+
+Install the **NuGet** package:
+
+Package Manager:
+
+```shell
+Install-Package Sentry -Version {{ packages.version('sentry.dotnet') }}
+```
+
+<div class="alert alert-info" role="alert"><h5 class="no_toc">Using .NET Framework prior to 4.6.1?</h5>
+    <div class="alert-body content-flush-bottom">
+        <a href="https://docs.sentry.io/clients/csharp/">Our legacy SDK</a> supports .NET Framework as early as 3.5.
+    </div>
+</div>
+
```

Comment:
I'm uncertain if this will render properly (and the preview of the wizard can be tricky) so may I suggest:

<Note>

If you are using .NET Framework prior to 4.6.1, our [legacy SDK](https://docs.sentry.io/clients/csharp/) supports .NET Framework as early as 3.5.

Reply:
I tested out and it works. We have that currently in Sentry for the .NET SDK